src/main/java/de/maxhenkel/voicechat/voice/common/Utils.java

```diff
@@ -107,12 +107,12 @@ public static short bytesToShort(byte b1, byte b2) {
         return stereo;
     }
 
-    public static Pair<Float, Float> getStereoVolume(Vector3d playerPos, float yaw, Vector3d soundPos) {
+    public static Pair<Float, Float> getStereoVolume(Vector3d playerPos, float yaw, Vector3d soundPos, float playerHeight, float talkingPlayerHeight, float maxDistance) {
         Vector3d d = soundPos.subtract(playerPos).normalize();
         Vector2f diff = new Vector2f((float) d.x, (float) d.z);
         float diffAngle = angle(diff, new Vector2f(-1F, 0F));
-
         float angle = normalizeAngle(diffAngle - (yaw % 360F));
+        float dif = playerHeight < talkingPlayerHeight ? Math.min((talkingPlayerHeight - playerHeight) / maxDistance, 1) : Math.min((playerHeight - talkingPlayerHeight) / maxDistance, 1);
```

Comment:
I think this line can be replaced with 
```suggestion
        float dif = Math.abs(playerHeight - talkingPlayerHeight) / maxDistance;
```
Please check this tho, I am writing just in a textbox ;)

Reply:
ooo good catch.